server/auth/store_test.go

```diff
@@ -991,3 +969,113 @@ func TestUserChangePasswordWithOldLog(t *testing.T) {
 		t.Fatalf("expected %v, got %v", ErrUserNotFound, err)
 	}
 }
+
+type backendMock struct {
```

Comment:
Maybe worth to put it in a separate file: store_mock_test.go as this file is already long and this might be a piece of intfra to use accross the files.

Reply:
Done.

Reply:
Done